kandroid/src/main/kotlin/com/pawegio/kandroid/KSeekBar.kt

```diff
@@ -20,16 +20,11 @@ import android.widget.SeekBar
 
 fun SeekBar.onProgressChanged(callback: (Int, Boolean) -> Unit) {
     setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
-        override fun onStopTrackingTouch(seekBar: SeekBar) {
+        override fun onStopTrackingTouch(seekBar: SeekBar) {}
```

Comment:
I would replace it with `override fun onStopTrackingTouch(seekBar: SeekBar) = Unit`